plutus-playground-client/src/Chain/BlockchainExploration.purs

```diff
@@ -180,34 +181,30 @@ toBalanceMap =
       where
         fromTxOut :: TxOutOf String -> Tuple (Tuple Column Row) Balance
         fromTxOut (TxOutOf { txOutType: (PayToPubKey (PubKey { getPubKey: owner }))
-                           , txOutValue: (Value ({ getValue: LedgerMap currencyBalances }))
+                           , txOutValue: (Value ({ getValue: currencyBalances }))
                            })
           = Tuple (Tuple (OwnerIx owner hash) row) (CurrencyBalance currencyBalances)
         fromTxOut (TxOutOf { txOutType: (PayToScript (DataScript { getDataScript: owner }))
-                           , txOutValue: (Value ({ getValue: LedgerMap currencyBalances }))
+                           , txOutValue: (Value ({ getValue: currencyBalances }))
                            })
           = Tuple (Tuple (ScriptIx owner hash) row) (CurrencyBalance currencyBalances)
 
+    -- | TODO Ask Jann if txFee is going to change away from `Ada` (the type) soon.
```

Comment:
Having the fees paid in anything other than Ada would be difficult because the slot leaders have to decide on what the currency should be. So it'll continue to be Ada.

Reply:
But the type will change, right? I thought `Ada`-the-type was going away...

Reply:
If we changed the `txFee` field to the `Value` type then we have to do one additional check when validating transactions (namely, that the `Value` only contains Ada). So I'd prefer to keep `Ada`-the-type for the fees. Or does that cause issues on your side?